templates/Salsa-CKAN-Extensions.cfn.yml

```diff
@@ -99,6 +99,14 @@ Parameters:
     Description: Branch/tag name in repository to checkout.
     Default: master
     Type: String
+  HarvestExtensionSource:
```

Comment:
hmm, i need to double check, i think this is now not in use.

Reply:
@ThrawnCA  is this now not in use?

Reply:
Correct, this file is no longer used. The only one that needs to be updated is `shared-OpenData.var.yml`

Reply:
should this file just be deleted in the PR?